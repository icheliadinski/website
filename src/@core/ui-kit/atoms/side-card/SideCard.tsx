import React, { ReactNode } from 'react'
import styles from './SideCard.module.scss'

export interface Props {
  children: ReactNode
}

export const SideCard: React.FC<Props> = ({ children }) => {
  return <section className={styles.card}>{children}</section>
}
