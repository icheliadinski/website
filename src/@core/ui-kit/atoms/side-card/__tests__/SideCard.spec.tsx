import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { SideCard, Props } from '../SideCard'

describe('[SideCard]', () => {
  let wrapper: ShallowWrapper<Props>

  beforeEach(() => {
    wrapper = shallow(<SideCard>content</SideCard>)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
