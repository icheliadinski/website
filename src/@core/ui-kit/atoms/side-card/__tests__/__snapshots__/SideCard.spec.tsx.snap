// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[SideCard] renders successfully 1`] = `
<section
  className="card"
>
  content
</section>
`;
